core/services/registrysyncer/local_registry.go

```diff
@@ -36,6 +36,22 @@ type LocalRegistry struct {
 	IDsToCapabilities map[string]Capability
 }
 
+func NewLocalRegistry(
+	lggr logger.Logger,
+	getPeerID func() (p2ptypes.PeerID, error),
+	IDsToDONs map[DonID]DON,
+	IDsToNodes map[p2ptypes.PeerID]kcr.CapabilitiesRegistryNodeInfo,
+	IDsToCapabilities map[string]Capability,
+) LocalRegistry {
+	return LocalRegistry{
+		lggr:              lggr,
```

Comment:
Should it add a name to the logger?

Reply:
This is only being used in tests as of now, but yea.